serverlessworkflow/modules/ROOT/pages/_common-content/getting-started-requirement.adoc

```diff
@@ -1,3 +1,3 @@
 * A workflow project is created.
 +
-For more information about creating a workflow project, see xref:getting-started/create-your-first-workflow-service.adoc[Creating your first Serverless Workflow service].
\ No newline at end of file
+For more information about creating a workflow project, see xref:use-cases/advanced-developer-use-cases/getting-started/create-your-first-workflow-service.adoc[Creating your first Serverless Workflow service].
```

Comment:
```suggestion
For more information about creating a workflow project, see xref:use-cases/advanced-developer-use-cases/getting-started/create-your-first-workflow-service.adoc[Creating your first Serverless Workflow Java service].
```

wdyt?

Reply:
Sure also created this to review everywhere - https://github.com/apache/incubator-kie-kogito-docs/issues/551